bedrock/base/templates/macros.html

```diff
@@ -482,6 +482,9 @@
             </li>
             <li><a href="{{ url('firefox.sync') }}"{% if activesub == 'sync' %} class="selected"{% endif %}><div>{{ _('Sync') }}</div></a></li>
             <li><a href="{{ url('firefox.hello') }}"{% if activesub == 'hello' %} class="selected"{% endif %}><div>{{ _('Hello') }}</div></a></li>
+            {% if l10n_has_tag('tracking_protection') and waffle.switch('tracking-protection') %}
```

Comment:
Remove the l10n tag check, only waffle is needed. We can't use tags on a shared thing like navigation.